### PR TITLE
V3 merge

### PR DESCRIPTION
**Pull request by patrickkettner**
@doctyper @SlexAxton @ryanseddon

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,51 +1,55 @@
 module.exports = function(grunt) {
 
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-nodeunit');
-  grunt.loadNpmTasks('grunt-contrib-watch');
+	grunt.loadNpmTasks('grunt-contrib-clean');
```
Comment (ryanseddon) [inline Gruntfile.js]: I'm on my phone but are these tabs? Can we do 2 spaces if so.

Comment (patrickkettner) [inline Gruntfile.js]: yeah, the rest of the project is in tabs though

Comment (SlexAxton) [inline Gruntfile.js]: Seems like a different PR

Comment (patrickkettner) [inline Gruntfile.js]: updated

**Comment by SlexAxton** [comment]
lol ok spaces!

**Comment by SlexAxton** [comment]
this looks pretty good. a ton of red is always a happy occasion.

**Comment by doctyper** [comment]
Hey this looks familiar! 👍

**Pull request merged**